src/RoslynPad/MainWindow.xaml.cs

```diff
@@ -161,6 +165,55 @@ private void LoadWindowLayout()
         {
             FontSize = _viewModel.Settings.WindowFontSize.Value;
         }
+
+        ShiftWindowOntoScreen();
+    }
+
+    private void GetScreenAndWindowInfo()
```

Comment:
debug info helper method, can be removed

Reply:
Please remove

Reply:
yeah, that was there mostly if you wanted to play with it. Removed.